### PR TITLE
fix: fin swap msg

### PR DESCRIPTION
**Pull request by samb-local**

### CODE REVIEW

File: contracts/packages/protocol/src/trade.rs [hunk 1]
```diff
@@ -62,9 +62,7 @@ pub struct SwapMsg {
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
-pub struct Swap {
-    pub offer_asset: Asset,
-}
+pub struct Swap {}
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 pub struct Asset {
```
Comment (davirds) [inline contracts/packages/protocol/src/trade.rs:68]: It seems that you are no longer using this struct, maybe you can delete it if you don't intend to use it in the future :)

**Review by davirds — CHANGES_REQUESTED** [review]
Just a little adjustment.

LGTM!

File: contracts/packages/protocol/src/trade.rs (hunk 1 above)
Comment (samb-local) [reply contracts/packages/protocol/src/trade.rs]: Good catch, fixed at https://github.com/Local-Money/localmoney/pull/64/commits/6f0cdb9346d52348f02c93fb8ec00747d161b2fc

**Review by davirds — APPROVED** [review]

**Review by davirds — APPROVED** [review]
LGTM!

**Review by davirds — APPROVED** [review]
LGTM

**Pull request merged**